packer/deploy-ami.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import sys
+import time
+
+import boto3
+
+logger = logging.getLogger(name=__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler(sys.stderr)
+logger.addHandler(handler)
+
+yaml_template ='''
+    {}:
+      '64': {}
+'''.strip('\r\n')
+
+parser = argparse.ArgumentParser(description='Copy an image to all available regions')
+parser.add_argument('-r', '--region', default='us-east-1')
+parser.add_argument('-i', '--ami-id', required=True)
+parser.add_argument('-q', '--quiet', action='store_true')
+
+def copy_to_region(image, region):
+    session = boto3.session.Session(region_name=region)
+    local_client = session.client('ec2')
+    logger.info("creating image in region {}".format(region))
+    resp = local_client.copy_image(
+        Name=image.name,
+        SourceImageId=image.image_id,
+        SourceRegion=args.region,
+        )
+    local_ec2 = session.resource('ec2')
+    new_image = local_ec2.Image(resp['ImageId'])
+    new_image.create_tags(Tags=image.tags)
+    
+    return (new_image, region)
+
+def wait_for_ready(image, region):
```

Comment:
this will wait forever. should we break if we have exceeded some threshold?

Reply:
yeah. What do you think a good timeout is?